fvwm/virtual.c

```diff
@@ -2383,7 +2388,7 @@ void CMD_GotoDeskAndPage(F_CMD_ARGS)
 	else
 		PeekToken(action, &action);
 
-	//free(action_cpy);
+	free(action_cpy_start);
```

Comment:
Oops.  Thanks for fixing this!

Reply:
`PeekToken` moves `action_cpy` forward a few bytes so I had to introduce another variable to be able to free the original pointer. Freeing `action_cpy` directly caused an Invalid free error from valgrind since it was pointing into the middle of the allocated memory instead of the beginning.

You're welcome.